src/components/FileBrowser/FileBrowser.vue

```diff
@@ -0,0 +1,252 @@
+<script>
```

Comment:
Interesting to do JS first, then template, then CSS. I don't mind this, but we should standardize on an ordering in Vue files for consistency.

Reply:
I picked this up at our local DC Vue meetup group.

The justification is that you always look at (html, script) or (html, css) but never (script, css), so it makes more sense to have your adjacency like this.

Reply:
I have been putting template, script, then style. but script, template, style actually feels better based on the frequency of the use. 
I learned a few style from the official style guide, https://vuejs.org/v2/style-guide/#Single-file-component-top-level-element-order-recommended

Reply:
I'll make a PR to add `plugin:vue/recommended` to our eslint extensions.

Reply:
#37 does this, but doesn't appear to include enforcement of the top level tag ordering... will look into this further after lunch.